UI/window-basic-main.cpp

```diff
@@ -5965,6 +5965,9 @@ void OBSBasic::StreamingStop(int code, QString last_error)
 
 void OBSBasic::AutoRemux()
 {
+	if (!outputHandler->lastRecordingPath.c_str())
+		return;
+
 	QString input = outputHandler->lastRecordingPath.c_str();
 	if (input.isEmpty())
 		return;
```

Comment:
Shouldn't we just replace this whole section of code with something like:
```cpp
	if (outputHandler->lastRecordingPath.empty)
		return;

	QString input = outputHandler->lastRecordingPath.c_str();
``` instead?